src/pyvism/vm.py

```diff
@@ -22,4 +22,4 @@ def run(self, bytecode: list[instruction[*tuple[Any, ...]]]) -> None:
 					file=sys.stderr,
 				)
 				if self.strict_mode:
-					return None
+					return
```

Comment:
Explicit is better than implicit -- PEP 8 recommends to write `return None`.

Reply:
Could you show me, please? As far as I'm aware, PEP8 wants you to be consistent with them:
> Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable)

This means that in this case, it was correct to have it as just `return`, since no other value is ever returned.

Reply:
> Could you show me, please? As far as I'm aware, PEP8 wants you to be consistent with them:
> 
> > Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable)
> 
> This means that in this case, it was correct to have it as just `return`, since no other value is ever returned.

It's my bad, I haven't stated anywhere about the roadmap. I intent to change the signature of this function in the future -- for now it always returns None, but it should not in my head.